libmproxy/flow.py

```diff
@@ -614,6 +619,43 @@ def killall(self, master):
         self.flows.kill_all(master)
 
 
+class Plugins(object):
```

Comment:
This looks a bit java-ish - what are your plans here? I think we can just add `ScriptContext.(add|remove)_contentview()` and don't need a Plugin registry thingie?

FWIW, I pondered with the idea of adding "extension"-functionality as well. My line of thought was using setuptools' `entry_points` so that you could `pip install mitmproxy-pig-contenview` to add sort of a permanent inline script that executes like a normal inline script (but is displayed differently in the UI). This is unrelated to this though - happy to discuss that in a separate place. :smiley: